static/css/style.css

```diff
@@ -1,18 +1,45 @@
-/* Sticky footer styles
--------------------------------------------------------------- */
+/* Custom page CSS
+-------------------------------------------------- */
+
+.login {
+	float:left;
+}
+
+.logged-in {
+	padding-top: 50px;
+}
+
+.container .credit {
+	margin: 20px;
+}
+
+.container {
+	margin-top: 20px;
+}
 
+#gbCountdown {
+	color: #FFFFFF;
+}
+
+code {
+	font-size: 80%;
+}
 html,
 body {
 	height: 100%;
-
-	/* The html and body elements cannot have any padding or margin. */
 }
 
 #wrap {
 	margin-top:110px;
 }
 
-/* Set the fixed height of the footer here */
+img{
```

Comment:
You don't want to change all images. Find the image that you're trying to set witdth 100px too and make it a class.